metricflow/model/data_warehouse_model_validator.py

```diff
@@ -340,25 +358,6 @@ def gen_measure_tasks(
                         error_message=f"Unable to query measure `{spec.element_name}` on data source `{data_source.name}` in data warehouse",
                     )
                 )
-
```

Comment:
@QMalcolm I think we don't actually need this step, as we are already querying for all the measures above one by one, this step seems to simply query them altogether which seems redundant. Unless there's another reason why we need it, then feel free to let me know and I'll add it back.

To iterate on that, what I mean is we already query 1 by 1 as followed in the above code,
```sql
SELECT measure1 FROM data_source; SELECT measure2 FROM data_source; SELECT measure3 FROM data_source;
```
then this next part we just do
```sql
SELECT measure1, measure2, measure2 FROM data_source
```

Reply:
NVM!

Reply:
Some extra context on this. Apologies for future people looking pack at this. Also, sorry that happened through a separate line of communication. I wasn't at my computer during this, and thus the details were communicated through slack at the time.

The big query is a short cut query. The queries for individual measures are run only if the short cut query fails. This is because we want to reduce the number of total queries run, thus if the short cut query passes, that means everything is okay and we don't need to run the individual queries. But if the short cut query fails, that means one or more of the items queried caused an issue. The key part is *one or more*. The error we get back from the data ware house will only be for the *first* queried item that failed. But, we want to create issues *for all* bad query items. Thus to find out how many and which of the queried items are bad we have to do the individual queries. This makes the happy path REALLY FAST. It does mean that in the worst case, wherein each data source has one one or more failed dimensions / measures / identifiers, *a lot* of queries get run. But this is to ensure we give complete information on all everything that is in a bad state.